### ISSUE TITLE
Review text color contrast

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/aurora#258]
Text color contrast throughout site does not meet WCAG 2.0 standards. WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. Level AAA requires a contrast ratio of at least 7:1 for normal text and 4.5:1 for large text  
[WebAIM Color Contrast Checker](https://webaim.org/resources/contrastchecker/)

Recommend using web accessibility tool like SiteImprove Accessibility Check or Wave to review.
